src/probnum/filtsmooth/gaussfiltsmooth/kalmanposterior.py

```diff
@@ -122,3 +121,31 @@ def __len__(self):
 
     def __getitem__(self, idx):
         return self.state_rvs[idx]
+
+    def sample(self, locations=None, size=()):
+        """
+        Draw samples from the filtering/smoothing posterior.
+
+
+        If location is None, it returns N joint samples from the list of RVs;
```

Comment:
If you end up changing something in the other docstring remember to also change it here.

Reply:
Well, I'd remove this one (so the one docstring from `FiltSmoothPosterior` is used). Alright?

Reply:
Yes, sounds good to me :+1: